machine/accton/accton_as4610_54/machine.make

```diff
@@ -0,0 +1,38 @@
+# Makefile fragment for Accton AS4610_54
+
+ONIE_ARCH ?= armv7a
+
+VENDOR_REV ?= 0
+
+# Translate hardware revision to ONIE hardware revision
+ifeq ($(VENDOR_REV),0)
+  MACHINE_REV = 0
+else
+  $(warning Unknown VENDOR_REV '$(VENDOR_REV)' for MACHINE '$(MACHINE)')
+  $(error Unknown VENDOR_REV)
+endif
+
+UBOOT_MACHINE = AS4610_54
```

Comment:
@lakesh-sharma -- that is an unlikely scenario as UBOOT contains ONIE specific patches.  ONIE cannot be ONIE without the ONIE u-boot patches.  What a platform might do, however, is add additional U-Boot patches on top of the ONIE u-boot patches.